Source/Core/VideoCommon/AsyncRequests.h

```diff
@@ -0,0 +1,90 @@
+// Copyright 2013 Dolphin Emulator Project
+// Licensed under GPLv2
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <atomic>
+#include <condition_variable>
+#include <mutex>
+#include <queue>
+
+#include "Common/CommonTypes.h"
+
+class AsyncRequests
```

Comment:
The class member have it, but the struct "member" of the Event lacks it. Do you want me to add it to this union struct as well?